README.md

```diff
@@ -69,7 +69,7 @@ This image required few file to be mounted in the container's `/data` folder.
 ```
 .aria2/session.txt  (empty file)
 .aria2/aria2.log    (empty file)
-.aria2/aria2.conf   (aria2 configuration file, not webui-aria2 conf)
+.aria2/aria2.conf   (aria2 configuration file, not webui-aria2 conf) must contains at least `enable-rpc=true` and `rpc-listen-all=true`

```

Comment:
Well, makes sense to me. My only concern was that it was fully usable I guess without the aria2.conf file, but now its compulsory.  As long as they are properly documented. Thanks @anthonyraymond !!